### PR TITLE
CQ-4351390 - File Attachment preview fix

### PR DESCRIPTION
**Pull request by TalmizAhmed**
## Description
Fix for File attachment preview.
Bug present in fileinputwidget. 
Also changed Prefill and Submit services to different names to avoid conflictwith v1 services.

## Related Issue
https://jira.corp.adobe.com/browse/CQ-4351390

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes and the overall coverage did not decrease.
- [ ] All unit tests pass on CircleCi.
- [ ] I ran all tests locally and they pass.

### CODE REVIEW

File: it/core/src/main/java/com/adobe/cq/forms/core/components/it/service/CustomAFSubmitService.java [hunk 1]
```diff
@@ -72,16 +67,16 @@ public Map<String, Object> submit(FormSubmitInfo formSubmitInfo) {
                         if (dataManager != null) {
                             String originalFileName = StringUtils.substringAfterLast(fileAttachment.getFileName(), "/");
                             String fileUrl = formSubmitInfo.getFormContainerPath() + "." + FileAttachmentServlet.SELECTOR + "/" + fileAttachmentUuid + "/" + originalFileName;
-                            fileAttachmentMapJsonObject = updateMapWithFileUrl(fileAttachmentMapJsonObject, fileAttachment.getFileName(), fileUrl);
-                            dataManager.put(fileAttachmentUuid, fileAttachment);
```
Comment (nit23uec) [inline it/core/src/main/java/com/adobe/cq/forms/core/components/it/service/CustomAFSubmitService.java:76]: does that means older custom prefill and submit service will not work with AF2 that used to work with AF1?

File: it/core/src/main/java/com/adobe/cq/forms/core/components/it/servlets/FileAttachmentServlet.java [hunk 2]
```diff
@@ -45,7 +45,7 @@
 @Component(service=Servlet.class,
         property={
                 "sling.servlet.methods=" + HttpConstants.METHOD_GET,
-                "sling.servlet.resourceTypes="+ "fd/af/components/guideContainer",
+                "sling.servlet.resourceTypes="+ "core/fd/components/form/container/v2/container",
                 "sling.servlet.selectors=" + FileAttachmentServlet.SELECTOR
         })
 @ServiceDescription("Adaptive Form File Attachment Retrieve Servlet")
```
Comment (nit23uec) [inline it/core/src/main/java/com/adobe/cq/forms/core/components/it/servlets/FileAttachmentServlet.java:51]: can't recall why its part of IT

File: it/core/src/main/java/com/adobe/cq/forms/core/components/it/service/CustomAFSubmitService.java (hunk 1 above)
Comment (TalmizAhmed) [reply it/core/src/main/java/com/adobe/cq/forms/core/components/it/service/CustomAFSubmitService.java:76]: @nit23uec The older was has been marked as old in cq guides [here](https://git.corp.adobe.com/livecycle/cq-guides/tree/master/it/aemds-guide-test-internal/src/main/java/com/adobe/aemds/guide/test/service)
The current one used in cq-guides is the implementation with a `List<FileAttachmentWrapper>` which now had to be used in core components as well and that's why it wasn't working.

File: it/core/src/main/java/com/adobe/cq/forms/core/components/it/servlets/FileAttachmentServlet.java (hunk 2 above)
Comment (TalmizAhmed) [reply it/core/src/main/java/com/adobe/cq/forms/core/components/it/servlets/FileAttachmentServlet.java:51]: @nit23uec I had the same question when I was trying to use `FormConstants` to import this path. Tbh I'm not sure why the Prefill and Submit services are a part of IT either.
@ravise5 Could you pitch in?

Comment (rismehta) [reply it/core/src/main/java/com/adobe/cq/forms/core/components/it/servlets/FileAttachmentServlet.java:51]: This is just a sample to test file attachment preview from server, its required

File: it/core/src/main/java/com/adobe/cq/forms/core/components/it/service/CustomAFSubmitService.java (hunk 1 above)
Comment (rismehta) [reply it/core/src/main/java/com/adobe/cq/forms/core/components/it/service/CustomAFSubmitService.java:76]: We don't support the deprecated file attachment map in the new servlets.

**Comment by rismehta** [comment]
@TalmizAhmed Build is failing, please check the logs and fix the PR

**Comment by rismehta** [comment]
Please check and fix this.

File: bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/models/form/package-info.java [hunk 3]
```diff
@@ -34,7 +34,7 @@
  * version, is bound to this proxy component resource type.
  * </p>
  */
-@Version("4.3.0")
+@Version("5.0.0")
```
Comment (rismehta) [inline bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/models/form/package-info.java:37]: Why have you bumped major version ?

Comment (TalmizAhmed) [reply bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/models/form/package-info.java:37]: CI suggested 5.0.0, should it be 4.4.0 then?

**Review by rismehta — APPROVED** [review]

File: it/core/pom.xml [hunk 4]
```diff
@@ -169,10 +168,8 @@ Import-Package: javax.annotation;version=0.0.0,*
         </dependency>
         <!-- AEM Forms SDK APi -->
         <dependency>
-            <groupId>com.adobe.aem</groupId>
+            <groupId>com.adobe.aemfd</groupId>
```
Comment (nit23uec) [inline it/core/pom.xml:171]: why this groupId is used?

**Pull request merged**